apps/api/pages/api/bookings/_post.ts

```diff
@@ -205,6 +207,13 @@ import { defaultResponder } from "@calcom/lib/server";
 async function handler(req: NextApiRequest) {
   const { userId, isAdmin } = req;
   if (isAdmin) req.userId = req.body.userId || userId;
+  const userIp = getIP(req);
+
+  await checkRateLimitAndThrowError({
```

Comment:
Why is this using "core" instead of "api"?

Reply:
We already have a global rate limit check by the API key

Reply:
`handleNewBooking` which was called from here was using core only. This is the reason.

Reply:
I think it makes sense to remove it then.

It also means that there were two different rate limitings in place for this route